denizen_scripts/survival/repo-link/global_systems/warps.dsc

```diff
@@ -132,29 +132,29 @@ warps_my_warps_GUI:
 warps_my_warps_GUI_open:
   type: task
   script:
-    - define inventory <inventory[warps_my_warps_GUI]>
-    - define type personal
-    - foreach <yaml[warps].list_keys[warps.personal].filter[starts_with[<player.uuid>]].include[<yaml[player.<player.uuid].read[warps.has_access]>]||<list>>]> as:identifier:
-      - inject build_warp_item
-      - if <[identifier].before[~]> == <player.uuid>:
-        - define list:|:<[item].with[nbt=action/warp].with[lore=<[item].lore.insert[<&e>ID<&co><&sp><&b><[identifier].after[~]>].at[1]>]>
-      - else:
-        - define list:|:<[item].with[nbt=action/warp]>
-    - if <[list]||null> != null:
-      - give <[list]> to:<[inventory]>
-    - inventory open d:<[inventory]>
+  - define inventory <inventory[warps_my_warps_GUI]>
+  - define type personal
+  - foreach <yaml[warps].list_keys[warps.personal].filter[starts_with[<player.uuid>]].include[<yaml[player.<player.uuid].read[warps.has_access]>]||<list>>]> as:identifier:
+    - inject build_warp_item
+    - if <[identifier].before[~]> == <player.uuid>:
+      - define list:|:<[item].with[nbt=action/warp].with[lore=<[item].lore.insert[<&e>ID<&co><&sp><&b><[identifier].after[~]>].at[1]>]>
+    - else:
+      - define list:|:<[item].with[nbt=action/warp]>
+  - if <[list]||null> != null:
```

Comment:
Something we may do here instead of checking if a ListTag isn't equal to `null` based on a fallback is checking whether the definition exists or not, or pre-defining `list` as an empty list before the `foreach` (`- define list <list>`), and checking `if <[list].is_empty>` to avoid over-using fallbacks